event/index.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+/**
+ * @ngdoc directive
+ * @name buildium.common.bdEvent
+ *
+ * @description
+ *
+ * Attach callbacks to arbitray DOM events.
+ *
+ * <!-- example -->

```

Comment:
`@example` to be parsed by doc generator

there are other tags available for [directives](https://github.com/angular/angular.js/wiki/Writing-AngularJS-Documentation#angularjs-specific-ngdoc-directives) as well